src/LmcCookieConsentManager.ts

```diff
@@ -87,19 +90,29 @@ const LmcCookieConsentManager: CookieConsentManager = (serviceName, args) => {
   } = options;
   const cookieName = 'lmc_ccm';
   const cookieConsent = window.initCookieConsent();
-
-  const languages = {
-    cs: configCs({ companyNames, ...translationOverrides.cs }, secondaryButtonMode, cookieTable.cs || {}),
-    de: configDe({ companyNames, ...translationOverrides.de }, secondaryButtonMode, cookieTable.de || {}),
-    en: configEn({ companyNames, ...translationOverrides.en }, secondaryButtonMode, cookieTable.en || {}),
-    hr: configHr({ companyNames, ...translationOverrides.hr }, secondaryButtonMode, cookieTable.hr || {}),
-    hu: configHu({ companyNames, ...translationOverrides.hu }, secondaryButtonMode, cookieTable.hu || {}),
-    pl: configPl({ companyNames, ...translationOverrides.pl }, secondaryButtonMode, cookieTable.pl || {}),
-    ru: configRu({ companyNames, ...translationOverrides.ru }, secondaryButtonMode, cookieTable.ru || {}),
-    sk: configSk({ companyNames, ...translationOverrides.sk }, secondaryButtonMode, cookieTable.sk || {}),
-    sl: configSl({ companyNames, ...translationOverrides.sl }, secondaryButtonMode, cookieTable.sl || {}),
-    uk: configUk({ companyNames, ...translationOverrides.uk }, secondaryButtonMode, cookieTable.uk || {}),
+  const languageConfigs = {
+    cs: configCs,
+    de: configDe,
+    en: configEn,
+    et: configEt,
+    hr: configHr,
+    hu: configHu,
+    lt: configLt,
+    lv: configLv,
+    pl: configPl,
+    ru: configRu,
+    sk: configSk,
+    sl: configSl,
+    uk: configUk,
   };
+  const languages = Object.entries(languageConfigs).reduce((acc, [code, configFunction]) => {
+    acc[code] = configFunction(
+      { companyNames, ...translationOverrides[code] },
+      secondaryButtonMode,
+      cookieTable[code] || {},
+    );
+    return acc;
```

Comment:
Please, consider adding a free line before `return` for better readability. Also, I suggest using a better name for accumulated values, e.g. what is in the variable.
```suggestion

    return overridedTranslations;
```